File: src/main/java/com/sedmelluq/discord/lavaplayer/player/event/PlayerPauseEvent.java
package com.sedmelluq.discord.lavaplayer.player.event;

import com.sedmelluq.discord.lavaplayer.player.AudioPlayer;

/**
 * Event that is fired when a player is paused.
 */
public class PlayerPauseEvent extends AudioEvent {
  /**
   * @param player Audio player
   */
  public PlayerPauseEvent(AudioPlayer player) {
    super(player);
  }
}

File: main/src/main/java/com/sedmelluq/discord/lavaplayer/manager/event/PlayerPauseEvent.kt
package com.sedmelluq.discord.lavaplayer.manager.event

import com.sedmelluq.discord.lavaplayer.manager.AudioPlayer

/**
 * Event that is fired when a player is paused.
 *
 * @param player Audio player
 */
data class PlayerPauseEvent(override val player: AudioPlayer) : AudioEvent
